internal/docstore/docstore.go

```diff
@@ -50,6 +50,15 @@ func newCollection(d driver.Collection) *Collection {
 	return &Collection{driver: d}
 }
 
+// RevisionField is the name of the document field used for document revision
+// information, to implement optimistic locking.
+// Every retrieved document will have this field set to a non-nil value
+// of unspecified type (different providers might use different types).
+// When defining a struct for storing docstore data, define the field to be of type
```

Comment:
What happens if the user doesn't do this?

Reply:
Clarified in the text (which has moved to the top of the file).